providers/fips/fipsprov.c

```diff
@@ -727,8 +727,12 @@ int fips_intern_provider_init(const OSSL_PROVIDER *provider,
 
     if ((*provctx = PROV_CTX_new()) == NULL)
         return 0;
-    PROV_CTX_set0_library_context(*provctx, c_get_libctx(provider));
-    PROV_CTX_set0_provider(*provctx, provider);
+    /*
+     * Only works because we are a built-in internal provider. This is not
```

Comment:
"Using the parent library context only works..."

Reply:
Done